packages/safe-tools-client/app/services/scheduled-payments-sdk.ts

```diff
@@ -46,6 +52,73 @@ export default class SchedulePaymentSDKService extends Service {
       this.contractOptions
     );
   }
+
+  @task *estimateExecutionGas(
+    moduleAddress: string,
+    tokenAddress: string,
+    amount: string,
+    payeeAddress: string,
+    maxGasPrice: string,
+    gasTokenAddress: string,
+    salt: string,
+    gasPrice: string,
+    payAt?: number | null,
+    recurringDayOfMonth?: number | null,
+    recurringUntil?: number | null
+  ): TaskGenerator<number> {
+    const scheduledPayments: ScheduledPaymentModule =
+      yield this.getSchedulePaymentsModule();
+    const result = yield scheduledPayments.estimateExecutionGas(
```

Comment:
I don't think the safe tool client needs to call this SDK function.

Reply:
👍